plugins/communications-app/InputForm/index.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import { Form, Container } from '@edx/paragon';
+
+const InputForm = ({
+  isValid, controlId, label, feedbackText,
+}) => {
```

Comment:
What do u think about this format:

```suggestion
const InputForm = ({
  isValid,
  controlId,
  label,
  feedbackText,
}) => {
```